engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -780,34 +766,27 @@ public RenderingStage getCurrentRenderStage() {
         return currentRenderingStage;
     }
 
+    /**
+     * Forces a recompilation of all shaders. This command, backed by Gestalt's monitoring feature,
+     * allows developers to hot-swap shaders for easy development.
+     *
+     * To run the command simply type "recompileShaders" and then press Enter in the console.
```

Comment:
By the way, I noticed that the command recompileShaders gives absolutely no feedback. There is no sense if the command actually did anything. Any chance we can print out to the console:

`Recompiling shaders.... (pause) done!`